src/modules/organisation/persistence/organisation.repository.ts

```diff
@@ -463,38 +463,36 @@ export class OrganisationRepository {
         return organisationEntity;
     }
 
-    public async setEnabledForItsLearning(
+    public async setEnabledForitslearning(
         personPermissions: PersonPermissions,
         id: string,
     ): Promise<DomainError | Organisation<true>> {
-        const permittedOrgas: PermittedOrgas = await personPermissions.getOrgIdsWithSystemrecht(
-            [RollenSystemRecht.SCHULEN_VERWALTEN],
-            true,
-        );
-        if (!permittedOrgas.all && permittedOrgas.orgaIds.length === 0) {
+        if (!(await personPermissions.hasSystemrechteAtRootOrganisation([RollenSystemRecht.SCHULEN_VERWALTEN]))) {
             return new EntityNotFoundError('Organisation', id);
         }
 
-        const organisationFound: Option<Organisation<true>> = await this.findById(id);
+        const organisationEntity: Option<OrganisationEntity> = await this.em.findOne(OrganisationEntity, id);
 
-        if (!organisationFound) {
+        if (!organisationEntity) {
             return new EntityNotFoundError('Organisation', id);
         }
-        if (organisationFound.typ !== OrganisationsTyp.SCHULE) {
+
+        if (organisationEntity.typ !== OrganisationsTyp.SCHULE) {
             return new EntityCouldNotBeUpdated('Organisation', id, [
                 'Only organisations of typ SCHULE can be enabled for ITSLearning.',
             ]);
         }
-        organisationFound.isEnabledForItsLearning = true;
+
+        organisationEntity.itslearningEnabled = true;
+        organisationEntity.version += 1;
```

Comment:
Might be useful when we create the GÜ for Schulen and we want to avoid unnecessary requests from Admin when the Schule is already in itsLearning... but then this endpoint should return a i18nkey as we usually do.